### ISSUE TITLE
Build fails in Vite

### ISSUE THREAD

**Issue by arpadgabor** [tldraw/signia#69]
Here's a reproduction. I tried excluding `signia-react-jsx/jsx-runtime` and the build indeed passes, but then there's a remaining import in the final bundle that points to nowhere and it crashes.

https://codesandbox.io/p/sandbox/damp-sky-7kh23d

Also discovered while creating the repro, for some reason dev mode works on local but it fails in codesandbox.

This is in dev mode in codesandbox:
![image](https://user-images.githubusercontent.com/9270494/229235030-85426b26-3604-479b-a39c-1f56972986bd.png)

And when building I get (on my local):

```
RollupError: "jsx" is not exported by "../node_modules/.pnpm/signia-react-jsx@0.1.0_react@18.2.0/node_modules/signia-react-jsx/jsx-runtime.mjs", imported by "src/components/layout/index.tsx
```

**Comment by ds300** [comment]
Thanks for the report! There are two issues here:

- the jsx-runtime.mjs file was still using common js exports 🤦🏼 , fixed in #70 
- the codesandbox has a typo in the vite config, `"react-signia-jsx"` instead of `"signia-react-jsx"`

**Comment by ds300** [comment]
Fix was published in v0.1.1 🙏🏼

**Comment by arpadgabor** [comment]
> the codesandbox has a typo in the vite config, "react-signia-jsx" instead of "signia-react-jsx"

ahh sorry, you're right 😅

thank you for the quick fix though 🙈
